.circleci/scripts/binary_linux_build.sh

```diff
@@ -11,7 +11,7 @@ NUM_CPUS=$(( $(nproc) - 2 ))
 # Defaults here for **binary** linux builds so they can be changed in one place
 export MAX_JOBS=${MAX_JOBS:-$(( ${NUM_CPUS} > ${MEMORY_LIMIT_MAX_JOBS} ? ${MEMORY_LIMIT_MAX_JOBS} : ${NUM_CPUS} ))}
 
-if [[ "${DESIRED_CUDA}" == "cu111" || "${DESIRED_CUDA}" == "cu113" ]]; then
+if [[ "${DESIRED_CUDA}" =~ cu11? ]]; then
```

Comment:
I don't know why `cu11?` nor `cu11\d` does not work, but `cu11[0-9]` does, but here it goes
```suggestion
if [[ "${DESIRED_CUDA}" =~ cu11[0-9] ]]; then
```